docs/x509/reference.rst

```diff
@@ -894,6 +894,17 @@ X.509 CSR (Certificate Signing Request) Object
         :raises UnicodeError: If an extension contains IDNA encoding that is
             invalid or not compliant with IDNA 2008.
 
+    .. method:: get_attribute_for_oid(oid)
```

Comment:
Note: I intend to add a doctest here in a subsequent PR, but I'm going to replace the existing CSR which will necessitate changing a few other doctest results so I wanted to keep that until after this lands.